linkerd/app/outbound/src/lib.rs

```diff
@@ -591,9 +584,7 @@ pub fn trace_labels() -> HashMap<String, String> {
 
 fn is_discovery_rejected(err: &Error) -> bool {
```

Comment:
this is now specifically for profiles, not for discovery --- maybe it needs a new name?

Reply:
also: nit/TIOLI: might be nice to refactor this  (and `is_loop`) into something more like `Rejected::matches` for consistency (definitely not a big deal though)

Reply:
oh, i see that we're getting rid of this in the next branch, nvm.